### PR TITLE
contest codes in cpp

### PR DESCRIPTION
**Pull request by Anuradha0501**

### CODE REVIEW

File: Contest - 1 - October 18 2020/cpp contest codes/cycleDetection.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * SinglyLinkedListNode {
+ *     int data;
+ *     SinglyLinkedListNode* next;
+ * };
+ *
+ */
+bool has_cycle(SinglyLinkedListNode* head) {
+    SinglyLinkedListNode *t = head;   // here t will point to head node
+SinglyLinkedListNode *r = head;
```
Comment (rkritika1508) [inline Contest - 1 - October 18 2020/cpp contest codes/cycleDetection.cpp:14]: Correct the indentation here. Make the code more readable.

Comment (Anuradha0501) [reply Contest - 1 - October 18 2020/cpp contest codes/cycleDetection.cpp:14]: ok

Comment (Anuradha0501) [reply Contest - 1 - October 18 2020/cpp contest codes/cycleDetection.cpp:14]: for correcting indentation again i had to make pr ?

Comment (rkritika1508) [reply Contest - 1 - October 18 2020/cpp contest codes/cycleDetection.cpp:14]: Naah, I have corrected it. Just make a note of it for future.

**Pull request merged**